src/ApiParametersDocument.js

```diff
@@ -69,17 +61,16 @@ export class ApiParametersDocument extends LitElement {
       transform: rotateZ(-180deg);
     }
 
-    .table-title {
-      white-space: nowrap;
-      overflow: hidden;
-      text-overflow: ellipsis;
+    .heading3 {
+      flex: 1;
+      color: var(--arc-font-subhead-color);
       font-size: var(--arc-font-subhead-font-size);
       font-weight: var(--arc-font-subhead-font-weight);
       line-height: var(--arc-font-subhead-line-height);
     }
-
-    :host([narrow]) .table-title {
-      font-size: var(--api-parameters-document-title-narrow-font-size, initial);
+    
+    :host([narrow]) .heading3 {
+      font-size: var(--arc-font-subhead-narrow-font-size, 17px);
```

Comment:
I don't really like this idea to put absolute values in the font-size. We started using rems long ago (not all components may be upgraded though). I don't remember the reason but there was a reason to put the `initial` as the default value.

Reply:
> Also, not to introduce another breaking change in the vars I would suggest using double default value for variables

Ok, then fallback variables also must be introduced in https://github.com/advanced-rest-client/api-headers-document/pull/9 and https://github.com/advanced-rest-client/api-body-document/pull/19.

> I don't really like this idea to put absolute values in the font-size. We started using rems long ago (not all components may be upgraded though).

Since I am not really aware of all changes, this is what I've seen in 4 similar components so far (in above-mentioned PRs). All these changes are related to these sections and I am only aware of this view.
![image](https://user-images.githubusercontent.com/6103913/111027635-743eb600-8402-11eb-8fbc-75050760a2d0.png)
In this case, `initial` will reset the font-size to 16px (browser's default value is 16px or 1em), and this is different from 17px set in other components. So if there's a good reason to keep `initial`, I'm okay with that as long as I can override it, but this doesn't seem to be ok for default styles.